hyperlane/isms/multisig/src/query.rs

```diff
@@ -166,51 +186,156 @@ mod tests {
         verify(ctx.as_immutable(), &message.encode(), &raw_metadata).should_fail();
     }
 
+    /// Test three scenarios:
+    ///
+    /// 1. Number of signatures is less than threshold.
+    /// 2. Number of signatures is greater than the size of validator set.
+    /// 3. A signature is from an unknown signer (who is not in the validator set).
     #[test]
-    fn rejecting_reuse_of_signature() {
```

Comment:
This test is deleted because it's covered by the new test, `rejecting_too_many_or_too_few_or_unknown_signatures`.